pages/index.js

```diff
@@ -1,10 +1,9 @@
 // --- React/Nextjs
-import { useEffect, useState, useCallback, useMemo } from 'react';
+import { useEffect, useState, useCallback, useMemo, useRef } from 'react';
+import Image from 'next/image';
 import Head from 'next/head';
```

Comment:
`index.js` and `lot/[id].js` will likely have conflicts w your other PRs but i can take care of them once the others are merged